manifests/testing/disks-images-provider.yaml.in

```diff
@@ -114,6 +114,27 @@ spec:
               - /ready
             initialDelaySeconds: 10
             periodSeconds: 5
+        - name: loopdev
+          command:
+          - sh
+          - -c
+          - |
+            while true; do
+              for i in $(seq 100); do
```

Comment:
This will start at 1, do we avoid 0 on purpose?

Reply:
Nope, adjusted to start at 0.